app/components/find_interface/courses/about_schools_component/view.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-l" id="section-schools"><%= course.placements_heading %></h2>
+  <div data-qa="course__about_schools">
+    <% if course.program_type == "higher_education_programme" && course.provider.provider_code != "B31" %>
```

Comment:
Can we stick this in a helper with a tests please

ie
```ruby
course.higher_education_programme?
```


`B31` looks vile, if that no longer exist can we remove please